### PR TITLE
MGMT-17948 Show number of devices selected by a new fleet

### PR DESCRIPTION
**Pull request by celdrake**
In the Create new fleet wizard, now the "device selector" will show the number of selected devices based on the label match.

https://github.com/flightctl/flightctl-ui/assets/829045/08d1dad8-34e6-4ad9-ad02-3ac5b4e52eed

### CODE REVIEW

**Comment by nirfarkas** [comment]
WOW! This looks great! Maybe we can make the number of selected devices more prominent? Let's brainstorm about this on Monday.

**Comment by celdrake** [comment]
> WOW! This looks great! Maybe we can make the number of selected devices more prominent? Let's brainstorm about this on Monday.

Thanks! Changed the message according to the new designs, to show an alert when there are added labels.

File: libs/ui-components/src/components/Fleet/CreateFleet/steps/DeviceLabelSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      <Alert isInline variant="danger" title={t('Failed to determine the number of selected devices')}>
+        {deviceCountError}
+        <Button
+          variant="link"
+          icon={<RedoIcon />}
+          onClick={() => {
+            reloadDeviceSelection(labels);
+          }}
+        >
+          Try again
```
Comment (rawagner) [inline libs/ui-components/src/components/Fleet/CreateFleet/steps/DeviceLabelSelector.tsx]: missing i18n

Comment (celdrake) [reply libs/ui-components/src/components/Fleet/CreateFleet/steps/DeviceLabelSelector.tsx]: Added

**Review by rawagner — APPROVED** [review]

**Pull request merged**